### DISCUSSION TITLE
Work with Alexa?

### DISCUSSION THREAD

**Discussion by Guruleenyc** [t0bst4r/home-assistant-matter-hub#354] [Q&A] [answered]
I have installed this add-on, added a bridge for Alexa on Port 5540 and included my light label. I then restarted this add-on and went to my Alexa app, but I don't see any new devices being detected on the Alexa side. What am I missing?

**Comment by Apollon77** [comment, marked as answer]
You will get an QR Code shown and then you use the Alexa app to add this bridge via this qr code
